flixel/system/debug/Console.hx

```diff
@@ -112,7 +112,7 @@ class Console extends Window
 		_input.addEventListener(FocusEvent.FOCUS_OUT, onFocusLost);
 		_input.addEventListener(KeyboardEvent.KEY_DOWN, onKeyPress);
 		
-		#if (cpp || neko) // workaround for broken TextField focus on native
```

Comment:
Hmm... I wonder if the legacy fix in that custom update function is the culprit. If we add the new conditional I wonder if it will "just work" in next.